.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!-- First time contributors: Take a moment to review CONTRIBUTING.md! -->
+<!-- Remove sections if not applicable -->
+## Summary of changes
+
+<!-- Summary goes here -->
+
+Closes <!-- issue number here -->
+
+### Pull Request Checklist
```

Comment:
Possible include a style check on the diff.  From the pandas template:

```
[ ] passes git diff upstream/master -u -- "*.py" | flake8 --diff
```

Reply:
For anything automated like that I'd rather just add it to the CI.

Reply:
Sure.  I'm a happy camper